frontend/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [2.0.543](https://github.com/open-ic/open-chat/releases/tag/v2.0.543-website)
```

Comment:
I think this should be surrounded by square brackets, so that the end result is itself surrounded by square brackets